subsys/bluetooth/host/iso.c

```diff
@@ -1383,7 +1383,14 @@ int bt_iso_big_terminate(struct bt_iso_big *big)
 void hci_le_big_complete(struct net_buf *buf)
 {
 	struct bt_hci_evt_le_big_complete *evt = (void *)buf->data;
-	struct bt_iso_big *big = &bigs[evt->big_handle];
+	struct bt_iso_big *big;
+
+	if (evt->big_handle > ARRAY_SIZE(bigs)) {
```

Comment:
Shouldn't this be `>=` (and in all the other places as well)?

Reply:
Yes, fixed that.

Reply:
@joerchan while your new change works, isn't it simpler to just do as suggested by Johan and do `>=` rather than `! <`?

Reply:
Sure, I just fixed this before I saw Johans comment.

I guess this is subjective, but in my head i find it easier to deal with negations of the expression rather than the opposite expression (as you can see by the fact I messed this up :P).
I can change to `>=` if that is preferred.

Reply:
`>=` is easier to read for me at least

Reply:
@jhedberg wrote:
> \>= is easier to read for me at least

@joerchan meaning, I'll wait for you to update to that before approving 😉